examples/FastAPI/fastapi_example_app.py

```diff
@@ -26,18 +25,25 @@ def read_root():
     return "Hello World!"
 
 
-example_app_public_address = socket.gethostbyname(socket.gethostname())
-example_app_address_as_seen_from_sba_container = "host.docker.internal"
+example_app_address = "host.docker.internal"
 example_sba_address = "localhost"
 
 pyctuator = Pyctuator(
     app,
     "Example FastAPI",
-    f"http://{example_app_public_address}:8000",
-    f"http://{example_app_address_as_seen_from_sba_container}:8000/pyctuator",
-    f"http://{example_sba_address}:8082/instances",
+    app_url=f"http://{example_app_address}:8000",
+    pyctuator_endpoint_url=f"http://{example_app_address}:8000/pyctuator",
+    registration_url=f"http://{example_sba_address}:8080/instances",
     app_description=app.description,
 )
 
-server = Server(config=(Config(app=app, loop="asyncio", host="0.0.0.0")))
+uvicorn_logger = logging.getLogger("uvicorn")
```

Comment:
Can we add a comment on why we're setting this logger level, please?

Reply:
sure, good idea